lib/eex/test/eex/tokenizer_test.exs

```diff
@@ -73,25 +73,19 @@ defmodule EEx.TokenizerTest do
   end
 
   test "improperly formatted quotation with interpolation" do
-    exprs = [
-      {:text, '<%% a <%= b %> c %>'}
-    ]
+    exprs = [{:text, '<%% a <%= b %> c %>'}]
```

Comment:
This is inconsistent with all other `exprs =` definition on the same file.

Reply:
That could/should be like the above list-wrapped kwlist and put the pair as this instead:
```elixir
exprs = [
  text: '<%% a <%= b %> c %>', # Always have trailing commas to make it easier to add/remove elements!
]
```